Makefile

```diff
@@ -154,6 +154,23 @@ endif
 	-t $(DOCKER_REPOSITORY):$(RESTY_IMAGE_BASE)-$(RESTY_IMAGE_TAG)-$(DOCKER_BASE_SUFFIX) . )
 	-$(UPDATE_CACHE_COMMAND) $(DOCKER_REPOSITORY):$(RESTY_IMAGE_BASE)-$(RESTY_IMAGE_TAG)-$(DOCKER_BASE_SUFFIX)
 
+build-golang:
+ifeq ($(RESTY_IMAGE_BASE),src)
+	@echo "nothing to be done"
+else
+	$(CACHE_COMMAND) $(DOCKER_REPOSITORY):golang-$(RESTY_IMAGE_BASE)-$(RESTY_IMAGE_TAG)-$(DOCKER_KONG_SUFFIX) || \
+	( $(MAKE) build-base ; \
+	$(DOCKER_COMMAND) -f dockerfiles/Dockerfile.golang \
+	--build-arg KONG_GO_PLUGINSERVER_VERSION=$(KONG_GO_PLUGINSERVER_VERSION) \
+	--build-arg RESTY_IMAGE_TAG="$(RESTY_IMAGE_TAG)" \
+	--build-arg RESTY_IMAGE_BASE=$(RESTY_IMAGE_BASE) \
+	--build-arg DOCKER_REPOSITORY=$(DOCKER_REPOSITORY) \
+	--build-arg DOCKER_BASE_SUFFIX=$(DOCKER_BASE_SUFFIX) \
+	--build-arg DEBUG=$(DEBUG) \
+	-t $(DOCKER_REPOSITORY):golang-$(RESTY_IMAGE_BASE)-$(RESTY_IMAGE_TAG)-$(DOCKER_KONG_SUFFIX) . )
+	-$(UPDATE_CACHE_COMMAND) $(DOCKER_REPOSITORY):golang-$(RESTY_IMAGE_BASE)-$(RESTY_IMAGE_TAG)-$(DOCKER_KONG_SUFFIX)
```

Comment:
the golang docker image will need to be regenerated if:
- the kong-build-tools sha changes
- the KONG_GO_PLUGINSERVER_VERSION variable changes

I think define a
```
DOCKER_GO_SUFFIX=${BUILD_TOOLS_SHA}-${KONG_GO_PLUGINSERVER_VERSION}${CACHE_BUSTER}
```
around line 85 and use it as the cache key (?)

Reply:
done